src/index.html

```diff
@@ -11,7 +11,122 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form>
+      <fieldset class="form__fieldset form-fieldset">
+        <legend class="form-fieldset__title">Personal information</legend>
+        <div class="field">
+          <label for="surname">Surname: </label>
+          <input type="text"
+            name="surname" 
+            required
+          >
+        </div>
+        <div class="field">
+          <label for="name">Name: </label>
```

Comment:
Add an 'id' attribute with the value 'name' to match the 'for' attribute of the label.